packages/network-controller/tests/provider-api-tests/shared-tests.ts

```diff
@@ -214,9 +227,8 @@ export const testsForProviderType = (providerType: ProviderType) => {
 
             await withNetworkClient(
               { providerType },
-              async ({ makeRpcCall, clock, blockTracker }) => {
+              async ({ makeRpcCall, blockTracker }) => {
                 await makeRpcCall(request);
-                await waitForNextBlockTracker(blockTracker, clock);
```

Comment:
@BelfordZ Do you remember? I feel like it might have something to do with https://github.com/MetaMask/web3-provider-engine/blob/cf612f898002833c36730d23972fe4c4dd483c76/index.js#L50, but I'm not sure.

Reply:
sry for the delay. 

We no longer need this because there was a subtle difference in the number of setTimeouts that occur between extension's network controller & the core one. [Specifically the use of 'stoplight'](https://github.com/MetaMask/web3-provider-engine/blob/main/index.js#L119) which [uses setTimeout if the lock is currently set](https://github.com/MetaMask/web3-provider-engine/blob/main/util/stoplight.js#L32). It only affected some tests based on whether or not the request would be made while the lock was held by another. Since we no longer are using this provider, this extra setTimeout goes away.